src/docs/rialto-servermanager-api.json

```diff
@@ -0,0 +1,314 @@
+          "$ref": "#/definitions/eventType"
+        }
+      },
+      "required": [
+        "eventType"
+      ]
+    },
+    "activateRequest": {
+        "type": "object",
+        "default": null,
+        "description": "",
+        "properties": {
+          "id": {
+            "$ref": "#/definitions/id"
+          }
+        },
+        "required": [
+          "id"
+        ]
+    },
+    "stopRequest": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/id"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "deactivateRequest": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/id"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "getStateRequest": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/id"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "registerRequest": {
+      "type": "object",
+      "properties": {
+        "event": {
+          "$ref": "#/definitions/eventType"
+        },
+        "id": {
+          "$ref": "#/definitions/eventId"
+        }
+      },
+      "required": [
+        "event",
+        "id"
+      ]
+    },
+    "unregisterRequest": {
+      "type": "object",
+      "properties": {
+        "event": {
+          "$ref": "#/definitions/eventType"
+        },
+        "id": {
+          "$ref": "#/definitions/eventId"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "activateResult": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "socket": {
+          "$ref": "#/definitions/socket"
+        }
+      },
+      "required": [
+        "socket"
+      ]
+    },
+    "stateResult": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/state"
+        }
+      },
+      "required": [
+        "state"
+      ]
+    },
+    "getListenersResult": {
+      "type": "object",
+      "properties": {
+        "listeners": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/listener"
+          }
+        }
+      },
+      "required": [
+        "listeners"
+      ]
+    }
+  },
+  "methods": {
+    "activate": {
+      "summary": "Activate an app session and spawn a socket",
+      "events": [
+        "StateChangeEvent"
+      ],
+      "params": {
+        "$ref": "#/definitions/activateRequest"
+      },
+      "result": {
+        "$ref": "#/definitions/activateResult"
+      },
+      "errors": [
+      ]
+    },
+    "stop": {
+      "summary": "Stops an app session and closes the socket",
```

Comment:
"Stops an app session and terminate the server.\n\nApp session server is moved to 'notrunning' state.\nIf App session server state changes, then StateChangeEvent 'notrunning' will be sent.\nIf App session server is already in 'notrunning' state, then method call will succeed but no StateChangeEvent will be sent."